Kozhemyachenok/example_15.py

```diff
@@ -0,0 +1,16 @@
+# 15 lines: itertools
+from itertools import groupby
```

Comment:
0 lines between import and code (1 line in another task). it should be consistent everywhere.